### PR TITLE
DHFPROD-5031: Adding load and mapping steps to flows

### PR DESCRIPTION
**Pull request by xnikhil08**
### Description
This PR contains changes for below mentioned tickets.
1. DHFPROD-5031: Add a load step to a new flow
2. DHFPROD-4749: Add a mapping step to a new flow

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Run.tsx [hunk 1]
```diff
@@ -20,8 +20,9 @@ const Statuses = {
     'FINISHED_WITH_ERRORS': 'finished_with_errors'
 }
 
-const Run: React.FC = () => {
-   const { resetSessionTime } = useContext(UserContext)
+// TODO Rename Bench component to Run
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/pages/Run.tsx:23]: Can remove comment

File: marklogic-data-hub-central/ui/src/components/load/load-card.tsx [hunk 2]
```diff
@@ -180,20 +194,27 @@ const LoadCard: React.FC<Props> = (props) => {
                             size="small"
                         >
                             <div className={styles.formatContainer}>
-                                <div style={sourceFormatStyle(elem.sourceFormat)}>{elem.sourceFormat.toUpperCase()}</div>
+                                <div style={sourceFormatStyle(elem.sourceFormat)} aria-label={`${elem.name}-sourceFormat`}>{elem.sourceFormat.toUpperCase()}</div>
                             </div>
-                            <div className={styles.stepNameStyle}>{getInitialChars(elem.name, 25, '...')}</div>
+                            <div className={styles.stepNameStyle} aria-label={`${elem.name}-stepName`}>{getInitialChars(elem.name, 25, '...')}</div>
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/load/load-card.tsx:199]: Are we adding a label here purely for testing? We dont allow duplicate step names. So do we need a lebel text.

File: marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx [hunk 3]
```diff
@@ -1,10 +1,118 @@
...
+        </AuthoritiesContext.Provider>
+      </MemoryRouter>
+    )
+
+    //Check if the card is rendered properly
+    expect(getByText('Add New')).toBeInTheDocument();
+    expect(getByText('testLoadXML')).toBeInTheDocument();
+    expect(getByLabelText('testLoadXML-sourceFormat')).toBeInTheDocument();
+    expect(getByText('Last Updated: 04/15/2020 2:22PM')).toBeInTheDocument();
+    expect(await (waitForElement(() => getByLabelText('testLoadXML-stepName')))).toBeInTheDocument();
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx:53]: any particular reason we are waiting for element here, there is no preceding fireEvent.

File: marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx [hunk 4]
```diff
@@ -1,10 +1,118 @@
...
+
+    fireEvent.mouseOver(getByText('testLoadXML')); // Hover over the Load Card to get more options
+
+    //Verify if the flow related options are availble on mouseOver
+    expect(getByTestId('testLoadXML-toNewFlow')).toBeInTheDocument(); // check if option 'Add to a new Flow' is visible
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx:58]: At any point only one step could be moused over to display "Add to New/Existing Flow" right. Do we need testid. We can just use getByText.

Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx:58]: Options "Add to New/Existing Flow" are available for every card in the document, but except for the one we hover at any point, all others are marked as 'display: none'. So thats why i could't use just the text "Add to New/Existing Flow" and had to provide them a proper unique test id.

File: marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx (hunk 3 above)
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/components/load/load-card.test.tsx]: It was added earlier but i guess i never removed it. Did it now as its not needed anymore.

File: marklogic-data-hub-central/ui/src/components/load/load-card.tsx (hunk 2 above)
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/components/load/load-card.tsx]: Yes, it was added only for testing purpose initially. But i don't think we need it anymore so have removed it.

File: marklogic-data-hub-central/ui/src/pages/Run.tsx (hunk 1 above)
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/pages/Run.tsx]: Done

File: marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx [hunk 5]
```diff
@@ -153,4 +162,120 @@ describe("Entity Tiles component", () => {
...
+
+    //Choose testFlow from the dropdown
+    fireEvent.click(getByText('testFlow'));
+    
+    //Click on 'Yes' button
+    fireEvent.click(getByTestId('Mapping1-to-testFlow-Confirm'));
+
+    //Check if the tiles-run route has been called
+    wait(() => {
+      expect(mockHistoryPush).toHaveBeenCalledWith('/tiles-run');
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-card.test.tsx:222]: Nice!
